python/GangaDirac/test/Unit/Backends/TestDiracBase.py

```diff
@@ -0,0 +1,422 @@
+import tempfile
+import os
+
+import pytest
+try:
+    from unittest.mock import patch, Mock
+except ImportError:
+    from mock import patch, Mock
+
+from Ganga.Core import GangaException
+from Ganga.GPIDev.Lib.Job import Job
+from Ganga.GPIDev.Lib.File import File
+from Ganga.GPIDev.Adapters.StandardJobConfig import StandardJobConfig
+from Ganga.Lib.Splitters import ArgSplitter
+from Ganga.Lib.Executable import Executable
+from GangaDirac.Lib.Backends import Dirac
+from GangaDirac.Lib.Backends.DiracBase import DiracBase
+from GangaDirac.Lib.Files.DiracFile import DiracFile
+
+script_template = """
+# dirac job created by ganga
```

Comment:
Can we please drop the hard-coded JDL template descriptions from the tests?
We want to know if the JDL is consistent to DIRAC but keeping around the template here is just adds to the burden whenever we need to change the API.